commands/anonSuggest.js

```diff
@@ -0,0 +1,19 @@
+require('discord.js');
```

Comment:
Unused require

Reply:
Pretty sure I took this from your code somewhere

Reply:
Nah all requires I've used pull something from it i.e `const {Attachment} = require('discord.js');`

Reply:
I've removed it now